main.py

```diff
@@ -44,3 +44,45 @@ def del_book_by_id(book_id: int):
         return {"Message": "Book doesn't exist in the library"}
 
 
+fake_db_authors = {}
+
+class Author(BaseModel):
+    id: int
+    name_author: str
+    surname_author: str
+    age_author: int
+    language_author: str
+    rating_author: float
+
+@app.get("/author")
+def all_authors():
+    all_authors = [fake_db_authors[key] for key in fake_db_authors.keys()]
+    return {"Authors": all_authors}
+
+@app.get("/author/{author_id}")
+def one_author(author_id: int):
+    if author_id in fake_db_authors:
+        return {"Author": fake_db_authors[author_id]}
+    else:
+        return {"Message": "Author doesn't exist in the library"}
+
+@app.post("/author")
```

Comment:
@IngElenaKozlova  what would happened if given id already exist in db? Looks old author would be replaced by a new one :)

Reply:
@Ilyusmax thank you, I'll fix it